.github/workflows/staging-dco.yml

```diff
@@ -0,0 +1,12 @@
+name: Staging DCO
+on:
+  push:
+    branches:
+      - staging
+
+jobs:
+  DCO:
+    runs-on: ubuntu-latest
+    steps:
+      - name: DCO
+        run: echo "DCO"
```

Comment:
Question: Is this a hack to get round DCO?

Reply:
Yes indeed, as DCO fails when bors has to rebase